tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -7336,6 +7339,49 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        plotInferenceEvolution: function() {
+          const container = Polymer.dom(this.$$('#inferenceEvolution'));
+          while (container.firstChild) {
+            container.removeChild(container.firstChild);
+          }
+          const example = this.selectedExampleAndInference;
+          if (
+            !example ||
+            !example.inferences ||
+            example.inferences.length < 2
+          ) {
+            return;
+          }
+          const inferences = example.inferences;
+          const isBinary = this.isBinaryClassification_(
+            this.modelType,
+            this.multiClass
+          );
+          const data = [];
+          for (let i = 0; i < inferences.length; i++) {
+            const inference = inferences[i];
+            for (let modelInd = 0; modelInd < inference.length; modelInd++) {
+              const modelInference = inference[modelInd];
+              for (
+                let labelInd = isBinary ? 1 : 0;
+                labelInd < modelInference.length;
+                labelInd++
+              ) {
+                const value = modelInference[labelInd].score;
+                if (!data[modelInd]) {
+                  data[modelInd] = {};
+                }
+                if (!data[modelInd][labelInd]) {
+                  data[modelInd][labelInd] = [{}];
+                }
+                data[modelInd][labelInd].push({step: i, scalar: value});
+              }
+            }
+          }
+          const chart = this.makeLineChart('Run', data, true);
+          container.appendChild(chart);
```

Comment:
we need to think harder about vertical space. with the chart and the table, the inference section takes up too much vertical space, leaving very little room for the datapoint viewing.

can we sync with mahima on this and perhaps have a control to switch between table view and chart view, instead of stacking?

Reply:
sure thing.